src/main/java/com/jcabi/aspects/aj/MethodValidator.java

```diff
@@ -148,18 +149,28 @@ public void after(final JoinPoint point, final Object result) {
      * @param object Object at pointcut
      * @param method Method at pointcut
      * @param args Parameters of the method
+     * @todo We need to find a way to disable the constraint rule
```

Comment:
@dmzaytsev todo has incorrect format, it should contain issue id and time estimate